RemoteInput/Plugin/Python/PythonShared.cxx

```diff
@@ -0,0 +1,1087 @@
+#include "PythonShared.hxx"
+#include "NativePlugin.hxx"
+
+// MARK: Non-EIOS
+
+PyObject* PythonCreateClient(PyObject* self, PyObject* args[], Py_ssize_t args_length) noexcept
+{
+    std::int32_t pid = from_python_object<std::int32_t>(args[0]);
+    EIOS_KillZombieClients();
+    EIOS_Inject_PID(pid);
+
+    auto found = false;
+    auto clients = EIOS_GetClients(true);
+    for (std::size_t i = 0; i < clients; i++)
+    {
+        auto client_pid = EIOS_GetClientPID(i);
+        if (client_pid == pid)
+        {
+            found = true;
+            break;
+        }
+    }
+
+    if (!found)
+    {
+        //(python->PyErr_Format)(python->PyExc_ProcessLookupError, "Unpaired client[%d] not found", pid);
```

Comment:
Returning `null` without throwing an exception will throw an internal exception. We probably want to throw our own? Or is this intentional?

Otherwise we should return `None` instead.

Reply:
I left a bunch of nullptr returns in here, I couldn't decide if it was worth defining some module exceptions and just used the nullptr as a placeholder to differentiate between the times were we do actually want to return none.